server/migrations/20181109171415-create-comment.js

```diff
@@ -0,0 +1,41 @@
+export default {
+  up: (queryInterface, Sequelize) => queryInterface.createTable('Comments', {
+    id: {
+      allowNull: false,
+      autoIncrement: true,
+      primaryKey: true,
+      type: Sequelize.INTEGER
+    },
+    body: {
+      allowNull: false,
+      type: Sequelize.STRING
+    },
+    userId: {
```

Comment:
I think this table should also have the name of the comment creator so that we do not have to query for that separately. Everything else looks good.

Reply:
couldn't we just eager-load that using the foreign key rather than adding another column?

Reply:
> I think this table should also have the name of the comment creator so that we do not have to query for that separately. Everything else looks good.

Yes, it it true. User ID will do that.

Reply:
> couldn't we just eager-load that using the foreign key rather than adding another column?

Thank you for the feedback. The userId on the model will serve the purpose. We do not need to create another column.

Reply:
On second thought, eager loading will do it.

Reply:
Okay. Thanks